keyboards/plywrks/ply8x/info.json

```diff
@@ -0,0 +1,237 @@
+{
+    "manufacturer": "plywrks",
+    "keyboard_name": "ply8x",
+    "maintainer": "ramonimbao",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "rows": ["B2", "B1", "B0", "A7", "A10", "A2"],
+        "cols": ["A9", "A8", "B12", "B11", "B10", "A6", "A5", "A4", "A15", "C13", "C14", "B9", "B7", "B6", "B5", "B4", "B3"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7905",
+        "vid": "0x706C"
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
+        "layout": [
+            {"flags": 8, "matrix": [3, 0], "x":2, "y":27},
+            {"flags": 8, "matrix": [0,15], "x":211, "y":0}
```

Comment:
> #define RGB_MATRIX_LED_COUNT 6

Only 2 entries here is invalid with the previous config

Reply:
Oops, sorry about that. Forgot to add the new LEDs after updating the number of LEDs.